apps/full-site-editing/full-site-editing-plugin/full-site-editing/utils/class-a8c-wp-template.php

```diff
@@ -33,7 +33,13 @@ class A8C_WP_Template {
 	 */
 	public function __construct( $post_id = null ) {
 		if ( null === $post_id ) {
-			$post_id = get_post()->ID;
+			$post = get_post();
+
+			if ( !$post ) {
```

Comment:
Nitpick: according to the [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage), we should put a space after the `!` operator 🙂

Reply:
Thanks, miguel! I shall fix. :)